---
layout: archive
title: "CV"
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}

Education
======
* PhD. Entomology (2016).  University of Georgia, Advisor: Dr. Michael J. Adang.
* M.Phil. Biochemistry (2011). University of Sussex, United Kingdom, Advisor: Dr. Neil Crickmore. 
* M.Sc. Genetic Manipulation and Molecular Cell Biology (2007). University of Sussex, Advisor: Dr. Neil Crickmore.

Work experience
======
* August 2016- Present: Post-Doctoral Associate
  * University of Florida
  
  * Project 1: Engineer pesticidal protein to improve its efficacy against Lepidopteran insect pest (Industry funded). 
  
   * Developed protocol for in vivo bio-panning assay against insects using phage display library.
   * Developed a pipeline to analyze Next Generation Sequencing in vivo bio-panning assay data.
   * Standardized UV-crosslink pull down assay for identification of binding partners of gut binding peptides.
   * Protein modelling and  modification.
   * Standardized MicroScale Thermophoresis (MST) assay for studying protein-protein interaction.

  * Project 2: Working as a team member for development of Bacterial Pesticidal Protein Resource Center.
      *  https://bpprc.org/  (under development)
   * Providing new nomenclature system for assigning names to bacterial pesticidal protein based on protein domain it possesses. 

  * PI: Dr. Bryony Bonning
  
Research Techniques
======
* Bottom-up proteomics: Sample preparation, Cell-surface protein enrichment, In-gel and In-solution digestion.
* Protein analytical techniques such as SDS-PAGE, Native PAGE, Western blot, Dot blot, ELISA, ion exchange, affinity and Size exclusion chromatography. 
  
* Peptide phage display library screening.
* Baculovirus expression of recombinant protein in insect cell lines.
* Confocal Microscopy. 
* Molecular biology techniques: Gene cloning, RNA and DNA extraction, PCR, cDNA synthesis. 

Bioinformatics and Programming Skills
======
* NGS RNA sequencing data analysis: reference genome based ((TopHat and Cufflinks) and de novo (Trinity), DESeq and Bioconductor.
* Single Nucleotide Variant analysis: VarScan, SnpEff.
 
* LC-MS/MS data analysis: Mascot, ProteoIQ, Trans Proteomic Pipeline and Proteome Discoverer.
* Next-generation sequencing of Phage display library and data analysis.

* R, Tidyverse (R package).
* Protein Modelling: PyMOL

Publications
======
  <ul>{% for post in site.publications %}
    {% include archive-single-cv.html %}
  {% endfor %}</ul>
  
